src/mavoscript.js

```diff
@@ -538,323 +520,11 @@ var _ = Mavo.Script = {
 		return [a, b];
 	},
 
-	childProperties: [
-		"arguments", "callee", // CallExpression
-		"left", "right", // BinaryExpression, LogicalExpression
-		"argument", // UnaryExpression
-		"elements", // ArrayExpression
-		"test", "consequent", "alternate", // ConditionalExpression
-		"object",  "property", // MemberExpression
-		"body"
-	],
+	walk: Vastly.walk,
```

Comment:
@DmitrySharabin what do you think? Should we continue to define these or just use vastly methods internally and print out a deprecation warning for these, directing plugin authors to use vastly too (using your [`Mavo.Functions.deprecatedFunction()`](https://github.com/mavoweb/mavo/blob/main/src/functions.js#L70))?

Reply:
I checked all Mavo plugins. Not only ours but also every plugin in the plugin registry. I couldn't find any that uses these `Mavo.Script` methods: `walk`, `closest`, and `serialize`. However, even though we don't use it in the plugins from the registry, we can't guarantee no (private) plugin does. So, I would follow the route with a deprecation message (until the next release minimum). At least we can lower the chance of breaking some author code and level up the awareness of vASTly, which is nice from my perspective. 😊